recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,3 @@
-# Ideally we would generate an OpenMPI build also, but the
-# build fails in that case (see #92).
+# Only supporting nompi because of python-mumps
```

Comment:
Any plans to add mpi support to python-mumps?

Reply:
Yes, absolutely. Should also be relatively straightforward since the wrapper is largely the same as Kwant. I didn't come around to doing so: very little experience with MPI an d not a high priority so far.